routines/all-documents.ini

```
@@ -0,0 +1,28 @@
+# Export with only the ID & a TITLE
+
+extension = js
+mimeType = application/json
+type = file
+label = min
```

It smells like a copy-paste.
```suggestion
label = all documents
```